fix-point/src/Rename.hs

```diff
@@ -17,36 +17,9 @@ TIO.putStrLn $ rename "CHANGE" "import Lib.App (AppEnv (..))\n\
 \import Servant.Server (serve)"
 :}
 -}
-changeLine :: Text -> Text -> Text
-changeLine newMod line = case T.words line of
-    -- module names
-    "module" : s : extra     -> T.unwords ("module": prefix s : extra)
-
-    -- imports which start with Lib
-    "import" : s : extra     -> if T.take libLen s == "Lib"
-                                then T.unwords ("import": prefix s : extra)
-                                else line
-
-    -- reexport where reexport module started with Lib
-    "(":"module" : s : extra -> T.unwords ("(" :"module": prefix s : extra)
-    ",":"module" : s : extra -> T.unwords ("," :"module": prefix s : extra)
-
-    -- Anything else can't be changed
-    _ -> line
-  where
-    prefix s = renamePrefix newMod (T.splitAt libLen s)
-
--- To use and avoid calling length function too many times
-libLen :: Int
-libLen = 3
-
--- Function replaces Lib prefix with given 1st argument
-renamePrefix :: Text -> (Text, Text) -> Text
-renamePrefix newMod (_, end) = newMod <> end
-
 -- Method to test if possible without filepath yet
 rename :: Text -> Text -> Text
-rename newMod s = T.unlines [changeLine newMod x | x <- T.lines s]
+rename newMod s = T.unlines [T.replace "Lib" newMod x | x <- T.lines s]
```

Comment:
I would like to keep previous implementation. `T.replace` will replace `Lib` everywhere in the module. If one day we will have function like `loadLibrary` then it will be unfortunate to have `loadOnwardsrary` at the end. The goal of manual pattern matching is that we want to change only exports and imports. Everything below this section shouldn't be touched.